sample_flask_code.py

```diff
@@ -0,0 +1,59 @@
+
+from flask import Flask,request,jsonify
+import json 
+import os
+
+app=Flask(__name__)
+
+data_file='data.json'
+
+@app.route('/tasks',methods=['POST'])
+def create_task():
+    new_task = request.get_json()
+
+    if not os.path.exists(data_file):
```

Comment:
This can be handled better, using try except block, Instead of creating file separately and then inserting data by opening the file again